src/main/osd/osd.c

```diff
@@ -631,6 +641,13 @@ static uint8_t osdShowStats(uint16_t endBatteryVoltage, int statsRowCount)
     }
 #endif
 
+#ifdef USE_RX_RSSI_DBM
+    if (osdStatGetState(OSD_STAT_MIN_RSSI_DBM)) {
+        itoa(stats.min_rssi_dbm, buff, 10);
+        osdDisplayStatisticLabel(top++, "MAX RSSI DBM", buff);
```

Comment:
This should be `MIN`, and be multiplied by -1 as well, to be consistent with the live display element.

Reply:
ok, that's strange ..  
 i know i did just that ... ?
 i had a  issue in the rebase ?  
 did i fix the wrong branch ...  ?
 ill review  all CODE  again when i get home ..  confused my self  this time 😒

Reply:
@mikeller  
code was unstaged other cosmetic changes space comments ect. were also in unstaged files .
sorry about that.... commit now includes what i tested .